n2survey/lime/survey.py

```diff
@@ -3,6 +3,8 @@
 
 import pandas as pd
 
+from n2survey import lime
```

Comment:
Doesn't it lead to the circular import issue? I.e. `from n2survey import lime` -(calls)-> `from .survey import *` -(calls)-> `from n2survey import lime`...

I would suggest importing directly `from .structure import read_lime_questionnaire_structure` to avoid recursive import problems.